components/elements-ui/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+export interface IButtonProps {
+  label: String
+  onClick?: (e:any) => void
+}
+const styles = {
+  border: '1px solid #eee',
+  borderRadius: 3,
+  backgroundColor: '#FFFFFF',
+  cursor: 'pointer',
+  fontSize: 15,
+  padding: '3px 10px',
+  margin: 10,
+};
+const Button: React.SFC<IButtonProps> = (props) => (
```

Comment:
Just a heads up here. `React.SFC` is deprecated. Not sure if your getting a warning here in vscode. Correct type is `React.FC`.